scripts/jest.config.js

```diff
@@ -0,0 +1,11 @@
+// @ts-check
+
+/**
+ * @type {jest.InitialOptions}
+ */
+module.exports = {
```

Comment:
added common setup to enable test search intellisense and typescript support

Reply:
yeah, we might complain, but it is a good thing to test these scripts